darts/models/theta.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 import statsmodels.tsa.holtwinters as hw
 
-from u8timeseries.utils.statistics import check_seasonality, extract_trend_and_seasonality, remove_seasonality
+from darts.utils.statistics import check_seasonality, extract_trend_and_seasonality, remove_seasonality
```

Comment:
did we want to unify imports?

Reply:
Yeah I think the plan was to make them all relative, if I'm not mistaken.

Reply:
done